specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/examples/ManagedInstanceCreateMax.json

```diff
@@ -25,7 +25,8 @@
         "publicDataEndpointEnabled": false,
         "proxyOverride": "Redirect",
         "timezoneId": "UTC",
-        "instancePoolId": "/subscriptions/20D7082A-0FC7-4468-82BD-542694D5042B/resourceGroups/testrg/providers/Microsoft.Sql/instancePools/pool1"
+        "instancePoolId": "/subscriptions/20D7082A-0FC7-4468-82BD-542694D5042B/resourceGroups/testrg/providers/Microsoft.Sql/instancePools/pool1",
+        "maintenancePolicyId": "/subscriptions/ab0e51c0-83c0-4380-8ae9-025516df392f/resourceGroups/Federation/providers/Microsoft.Maintenance/maintenanceConfigurations/MiPolicy1"
```

Comment:
>maintenancePolicyId [](start = 9, length = 19)

Is this a required property?

Reply:
This property is optional. Resolved in an email.